### PR TITLE
How to update 1click app 

### LINKED ISSUE #1388: Add paragraph on how to update 1click
https://doc.scalingo.com/platform/deployment/one-click-deploy

### PR DESCRIPTION
**Pull request by btrd**
fix #1388

### CODE REVIEW

File: _posts/platform/deployment/2000-01-01-one-click-deploy.md [hunk 1]
```diff
@@ -65,3 +65,17 @@ Example:
 ```
 
 You can find everything about the `scalingo.json` file on our [API dedicated page](http://developers.scalingo.com/scalingo-json-schema/).
+
+## Update the application
```
Comment (EtienneM) [inline _posts/platform/deployment/2000-01-01-one-click-deploy.md:69]:
```suggestion
## Update the Application
```

File: _posts/platform/deployment/2000-01-01-one-click-deploy.md [hunk 2]
```diff
@@ -65,3 +65,17 @@ Example:
...
+
+To update your application to the latest version of your code, you can use an archive. Github is giving you access to an archive with this URL.
```
Comment (EtienneM) [inline _posts/platform/deployment/2000-01-01-one-click-deploy.md:71]:
```suggestion
To update your application to the latest version of your code, you can use an archive. GitHub is giving you access to an archive with this URL:
```

File: _posts/platform/getting-started/2000-01-01-getting-started-with-metabase.md [hunk 3]
```diff
@@ -41,3 +41,13 @@ Last, deploy your application with:
 ```bash
 $ git push scalingo master
 ```
+
+## Update the application
```
Comment (EtienneM) [inline _posts/platform/getting-started/2000-01-01-getting-started-with-metabase.md:45]:
```suggestion
## Update the Application
```

File: _posts/platform/getting-started/2000-01-01-getting-started-with-metabase.md [hunk 4]
```diff
@@ -41,3 +41,13 @@ Last, deploy your application with:
...
+
+To update your Metabase application to the latest version, you simply need to redeploy the last version of the code source.
```
Comment (EtienneM) [inline _posts/platform/getting-started/2000-01-01-getting-started-with-metabase.md:47]: Nothing is "simple". This is one of our advice when writing documentation: https://www.notion.so/scalingooriginal/Writing-Documentation-at-Scalingo-bb7985d6ea16455894417e36ede09cea

```suggestion
To update your Metabase application to the latest version, you need to redeploy the last version of the code source.
```

**Review by EtienneM — CHANGES_REQUESTED** [review]

File: _posts/platform/getting-started/2000-01-01-getting-started-with-metabase.md (hunk 4 above)
Comment (btrd) [reply _posts/platform/getting-started/2000-01-01-getting-started-with-metabase.md]: True!

Thanks for your feedback !

**Review by EtienneM — APPROVED** [review]

**Pull request merged**